app/configs/chargeColumns.tsx

```diff
@@ -1,21 +1,21 @@
 import { ColDataCardDef } from "@/types/types";
 import { Charge } from "@/types/charge";
 
-export const ALL_CHARGE_COLUMNS: { [name: string]: ColDataCardDef<Charge> } = {
+export const ALL_CHARGE_COLUMNS = {
   unit_nbr: {
-    name: "unit_nbr",
+    path: "unit_nbr",
     label: "Unit",
   },
   prsn_nbr: {
-    name: "prsn_nbr",
+    path: "prsn_nbr",
     label: "Person",
   },
   charge: {
-    name: "charge",
+    path: "charge",
     label: "Charge",
   },
   citation_nbr: {
-    name: "citation_nbr",
+    path: "citation_nbr",
     label: "Citation Number",
   },
-};
+} satisfies Record<string, ColDataCardDef<Charge>>;
```

Comment:
[here are the docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html) about Typescript satisfies. it seems like it was designed for this exact scenario, it lets TypeScript infer this object's keys while still checking that column config matches the `ColDataCardDef`

this change fixes a type bug down in `crashDataCard` 👇